bundle/manifests/tackle-operator.clusterserviceversion.yaml

```diff
@@ -78,7 +78,7 @@ spec:
 
     ### Install
 
-    Tackle requires a storage class that supports RWX volumes, please refer to the [documentation](https://konveyor.github.io/tackle/installation/) for system requirements details.
+    By default, Tackle requires a RWX storage class for caching purposes, this can be turned off via the rwx_supported parameter in Tackle CR.
```

Comment:
perhaps `disabled` instead of `turned off`?

Reply:
Yes much better, updated phrasing.